ir/ir.h

```diff
@@ -17,41 +17,8 @@ limitations under the License.
 #ifndef _IR_IR_H_
 #define _IR_IR_H_
 
-#include <strings.h>
```

Comment:
Most of the IR includes have been removed and only `ir-inline.h` and `ir-generated.h` remain. Hopefully, this reduces the amount of megaincludes in the compiler.